src/settings/App.svelte

```diff
@@ -157,8 +160,13 @@
     bind:value={form.message}
   />
   <AdditionalSettings bind:settings={form.additionalSettings} />
-  <div>
-    <vscode-button on:click={createConfFile}>{submitButtonText}</vscode-button>
+  <div class="save-button">
+    <vscode-button
+      on:click={createConfFile}
+      disabled={!form.mainSolidityFile ||
```

Comment:
That's interesting how this view is rendered. If you fill all the mandatory fields but then clear the main contract input, the button does not become disabled again.

Reply:
you are right! I'll change it